workspaces/marketplace/plugins/catalog-backend-module-marketplace/src/module.ts

```diff
@@ -33,11 +35,16 @@ export const catalogModuleMarketplace = createBackendModule({
       deps: {
         logger: coreServices.logger,
         catalog: catalogProcessingExtensionPoint,
+        discovery: coreServices.discovery,
       },
-      async init({ logger, catalog }) {
-        logger.info('Marketplace provider initialized!');
+      async init({ logger, catalog, discovery }) {
+        logger.info('Marketplace module initialized!');
```

Comment:
It's not initialized yet. It's initializing... :)

```suggestion
        logger.info('Adding Marketplace processors to catalog...');
```